utils/pack.ts

```diff
@@ -0,0 +1,52 @@
+import execa from 'execa';
+import path from 'path';
+import fs from 'fs-extra';
+import { TurboDryRun } from './types';
+
+const rootDir = path.join(__dirname, '..');
+
+async function main() {
+  const { stdout: sha } = await execa('git', ['rev-parse', '--short', 'HEAD'], {
+    cwd: rootDir,
+  });
+  const { stdout: turboStdout } = await execa(
+    'turbo',
+    ['run', 'build', '--dry=json'],
+    {
+      cwd: rootDir,
+    }
+  );
+  const turboJson: TurboDryRun = JSON.parse(turboStdout);
+  for (const task of turboJson.tasks) {
+    const dir = path.join(rootDir, task.directory);
+    const packageJsonPath = path.join(dir, 'package.json');
+    const originalPackageObj = await fs.readJson(packageJsonPath);
+    const packageObj = await fs.readJson(packageJsonPath);
```

Comment:
Do we want to read this file twice or clone the object? I doubt it's a perf issue to read twice. So, either is fine with me. It just seemed a bit weird on an initial read.

Reply:
Hmm, that does seems strange to read it twice. Although if we need a deep clone, then there isn't a nice way to do that without a 3rd party package (or Node.js 17+ structuredClone)

Reply:
Happy to clone it instead